popular-crates.txt

```diff
@@ -1,120 +1,208 @@
-# To re-generate this list, run:
-#     curl https://lib.rs/command-line-utilities > cli.html
-#     cat cli.html |
-#         pup ':parent-of(:parent-of(:parent-of(.bin))) json{}' |
-#         jq -r '.[] |
-#             (.children[1].children|map(select(.class == "downloads").title)[0]// "0 ")
-#             + ":" +
-#             (.children[0].children[0].text)' |
-#         sort -gr |
-#         grep -v / |
-#         head -n 100 |
-#         sed s/^.*:// |
-#         sort
-# and then audit any new entries. Crates from contributors go to the top of the list :D.
+# To re-generate this list, run ./rebuild-popular-crates.sh and then audit any new entries.
+# Crates from contributors go to the top of the list :D.
 #
 # The first empty line is important.
 
 cargo-quickinstall
 sensei
 
 b3sum
-bacon
 bandwhich
 bat
+bingrep
 bottom
 broot
-cargo-binutils
-cargo-bump
+capter
+cargo-about
+cargo-all-features
+cargo-apk
+cargo-asm
+cargo-audit
+cargo-aur
+cargo-bitbake
+cargo-bloat
+cargo-bom
+cargo-bpf
+cargo-build-deps
+cargo-bundle
+cargo-c
+cargo-cache
+cargo-check
+cargo-chef
+cargo-clean-recursive
+cargo-clone
+cargo-clone-crate
+cargo-compete
+cargo-crev
+cargo-criterion
+cargo-deadlinks
+cargo-deb
+cargo-deny
+cargo-dephell
+cargo-deps
+cargo-download
+cargo-ebuild
+cargo-edit
+cargo-equip
+cargo-geiger
+cargo-hack
+cargo-index
+cargo-info
+cargo-insta
+cargo-instruments
+cargo-kcov
+cargo-license
+cargo-lichking
+cargo-lipo
+cargo-llvm-lines
+cargo-local-registry
+cargo-make
+cargo-modules
+cargo-ndk
+cargo-outdated
+cargo-pants
+cargo-pgx
+cargo-pkg
+cargo-play
+cargo-profiler
+cargo-raze
+cargo-readme
+cargo-release
 cargo-rpm
+cargo-script
+cargo-show
+cargo-skyline
+cargo-sort-ck
+cargo-spellcheck
+cargo-strip
+cargo-subcommand
+cargo-sweep
+cargo-sync-readme
+cargo-tarpaulin
+cargo-task
+cargo-travis
+cargo-tree
+cargo-trend
 cargo-trim
+cargo-unleash
+cargo-update
+cargo-updater
+cargo-upgrades
+cargo-ux
+cargo-valgrind
+cargo-vcpkg
+cargo-vendor
+cargo-version-sync
+cargo-vs
+cargo-wasi
+cargo-watch
+cargo-web
+cargo-wgsl
+cargo-whatfeatures
+cargo-when
+cargo-why
+cargo-with
+cargo-wix
+cargo-xbuild
 cicada
-cobalt-bin
-code-minimap
 complate
 critcmp
-dijo
-diskus
-dmenv
-dotenv-linter
+cross
+diener
+dinero-rs
+dors
+dotter
 dtn7
+duct
 du-dust
 dutree
 elf2tab
+emils-cargo-release
+emplace
+error_show
 exa
-fblog
 fd-find
 feroxbuster
-ff-find
 ffsend
-file-sniffer
-flowy
+flamegraph
 fw
 git-delta
 git_lab_cli
+git-state
 gitui
 gptman
 grex
+hcal
 hexyl
-hunter
 hyperfine
 i2p_client
 intel-mkl-tool
 investments
+itm
 jql
+json2struct
+json_diff
 just
 kickstart
+kmon
+kubie
+kurobako
+landon
+leetcode-cli
 lsd
-mdblog
+lts
+macchina
 mdcat
 my-iot
+natls
+navi
 nitrocli
 nj-cli
-otpcli
+odbcsv
+pastel
 petname
-phetch
-podcast
 procs
 project_init
 pueue
+pyoxidizer
+regress
+rerast
 rexcli
-ripgrep
```

Comment:
Ooh that's interesting: ripgrep isn't in the popular crates list anymore. I guess most people install it from their operating system package manager instead?

Maybe I will add it to the list at the top layer, so that my readme doesn't need changing.

Reply:
lol didn't think about that.